resources/users/users.model.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+const UserSchema = Schema({
+  first_name: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 50
+  },
+  last_name: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 80
+  },
+  email: {
+    type: String,
+    required: true,
+    minlength: 2,
+    maxlength: 80,
+    unique: true
+  },
+  password: {
+    type: String,
+    required: true,
+    minlength: 5,
+    maxlength: 80
+  }
+});
```

Comment:
I think it would be good to have username here so as to make user login using their `username`

Reply:
ok, will do that